dask/dataframe/io/tests/test_orc.py

```diff
@@ -97,7 +97,7 @@ def test_orc_roundtrip(tmpdir, index, columns):
         }
     )
     if index:
-        data.set_index(index, inplace=True)
+        data = data.set_index(index)
```

Comment:
Note that here, and in other tests, our use of `inplace=True` isn't actually needed. All these `pandas` DataFrames are small, so I've just removed the `inplace` option altogether.